### ISSUE TITLE
Run OpenTrack remotely (e.g. on a laptop w/ webcam) while running AITrack locally (e.g. on a desktop w/o webcam)

### ISSUE THREAD

**Issue by GamerKingFaiz** [AIRLegend/aitrack#46]
**Describe your idea**
I'd to be able to run OpenTrack remotely on my laptop that has a webcam. Then I'd like AITrack to be able to hook into the OpenTrack feed.

**Describe the solution you'd like**
It looks like network capabilities are already built into both (you have to specify an IP address and port), but I'm not sure if there are any other limiting factors.

**Additional context**
Currently when I open AITrack on my desktop it will error out saying that there's no webcam connected. And the options are greyed out to input my laptop's IP address (can't even tick the checkbox).
![image](https://user-images.githubusercontent.com/9065136/92314023-15960c80-ef87-11ea-9f26-71c29d77bbe2.png)

**Comment by AIRLegend** [comment]
You should open AITrack on your laptop and then put the set the "Use remote Opentrack" fields with the machine running Opentrack (your desktop in this case). AITrack will then send the data via UDP to your main computer.

**Comment by GamerKingFaiz** [comment]
Oh yeah, I guess I had the order backwards. That did it--thanks!

**Comment by pedro-1952** [comment]
Hey I am having problems setting the port number. Which port did you use?

**Comment by GamerKingFaiz** [comment]
I don't remember because it's been awhile and I ended up using a local webcam. But I'm pretty sure I just used the opentrack default, which seems to be `4242`. Make sure you're using the same port in AITrack too.

Otherwise make sure it's not a firewall issue or some other reason the port isn't working. Maybe post screenshots of your settings and any errors you get if you still can't get it work.
